src/integration-test/groovy/de/iteratec/osm/measurement/environment/wptserver/PersistingResultsIntegrationSpec.groovy

```diff
@@ -131,12 +177,22 @@ class PersistingResultsIntegrationSpec extends NonTransactionalIntegrationSpec {
                 label: 'FF_BV1_Multistep_2',
                 location: loc
         )
+
+        Job.build(
+                label: '1Run_11events_4Faulty',
+                location: loc
+        )
     }
 
     void createMocksCommonToAllTests() {
         TimeToCsMappingService timeToCsMappingService = Stub(TimeToCsMappingService)
         timeToCsMappingService.getCustomerSatisfactionInPercent(_, _, _) >> 42
         resultPersisterService.timeToCsMappingService = timeToCsMappingService
+
+        resultPersisterService.configService = Stub(ConfigService) {
```

Comment:
Please restore the original configService in the `cleanup` method to avoid test data pollution.